packages/jest-environment-webdriver/modules/WebDriverEnvironment.js

```diff
@@ -6,14 +6,17 @@ class WebDriverEnvironment extends NodeEnvironment {
     super(config);
     const options = config.testEnvironmentOptions || {};
     this.browserName = options.browser || 'chrome';
+    this.usingServer = options.usingServer || null;
   }
 
   async setup() {
     await super.setup();
-
-    const driver = await new Builder()
-      .forBrowser(this.browserName)
-      .build();
+    
+    let driver = await new Builder();
```

Comment:
You can avoid `await` here, since there is no `build()` in the chain yet.

Reply:
Thank you!